03/app.js

```diff
@@ -1,3 +1,10 @@
 export default function randomNumber(min, max) {
-
-}
\ No newline at end of file
+  if (min === 1 && max === 1) return 1;
+  if (typeof min !== 'number' || typeof max !== 'number') {
```

Comment:
Pamiętaj, aby sprawdzić czy nie jest `NaN` - to wartość która jest jest `number`